documentation/book/appendix_metrics.adoc

```diff
@@ -37,66 +37,160 @@ endif::Kubernetes[]
+To create this `Secret` resource, use the following command:
 
 [source,shell,subs=attributes+]
-curl -s https://raw.githubusercontent.com/strimzi/strimzi-kafka-operator/{GithubVersion}/metrics/examples/prometheus/alerting-rules.yaml > alerting-rules.yaml
+curl -s https://raw.githubusercontent.com/strimzi/strimzi-kafka-operator/{GithubVersion}/metrics/examples/prometheus/additional-properties/prometheus-additional.yaml
+oc create secret generic additional-scrape-configs --from-file=prometheus-additional.yaml
+
+The provided `prometheus-rules.yaml` file creates a `PrometheusRule` with sample alerting rules. Download and update the resource file as follows:
 
-To deploy all these resources you can run the following.
+[source,shell,subs="+quotes"]
+curl -s https://raw.githubusercontent.com/strimzi/strimzi-kafka-operator/{GithubVersion}/metrics/examples/prometheus/install/prometheus-rules.yaml | sed -e 's/namespace: .*/namespace: _my-namespace_/' > prometheus-rules.yaml
+
+To deploy these resources, run the following commands:
 
 [source,shell,subs=attributes+]
 oc login -u system:admin
-oc apply -f alerting-rules.yaml
+oc apply -f strimzi-service-monitor.yaml
+oc apply -f prometheus-rules.yaml
 oc apply -f prometheus.yaml
 
+Prometheus also provides an alerting system through the link:https://prometheus.io/docs/alerting/alertmanager/[Alertmanager^] component.
+To enable alerting, the provided link:https://raw.githubusercontent.com/strimzi/strimzi-kafka-operator/{GithubVersion}/metrics/examples/prometheus/install/prometheus-rules.yaml[`prometheus-rules.yaml`^] file describes a `PrometheusRule` resource that defines sample alerting rules for Kafka and Zookeeper metrics.
+When an alert condition is evaluated as true on the Prometheus server, it sends the alert data to the Alertmanager which then uses the configured notification methods to notify the user.
+
+For more information about setting up alerting rules, see https://prometheus.io/docs/prometheus/latest/configuration/alerting_rules/[Alerting Rules] in the Prometheus documentation.
+
 ifdef::Kubernetes[]
-==== Deploying on {KubernetesName}
+==== Deploying Prometheus on {KubernetesName}
 
 The provided `prometheus.yaml` file, with all the Prometheus related resources, creates a `ClusterRoleBinding` in the `myproject` namespace.
+It also discovers an Alertmanager instance in the same namespace.
 If you are using a different namespace, download the resource file and update it as follows:
 
+[source,shell,subs="+quotes"]
+curl -s https://raw.githubusercontent.com/strimzi/strimzi-kafka-operator/{GithubVersion}/metrics/examples/prometheus/install/prometheus.yaml | sed -e 's/namespace: .*/namespace: _my-namespace_/' > prometheus.yaml
```

Comment:
I think that on other places we have Linux and MacOS versions. Should we have them here as well?